authenticator/views/application-list.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { StyleSheet, View, FlatList } from 'react-native';
+import ApplicationListItem from "../components/application-list-item"
+
+let example_data =
+    [
+        '000000',
+        '000001',
+        
+    ];

```

Comment:
We should abstract this out and put a comment so we know there needs to be permanent data storage linked here

Reply:
It's only there for visualization purposes because there is no backend yet.

Reply:
That makes sense, I think we should remove it before merging to main